.github/workflows/ci.yml

```diff
@@ -69,9 +92,27 @@ jobs:
         if:   always()
         run:  docker-compose logs
 
+  integration-tests:
+    runs-on: ubuntu-latest
+    steps:
       - name: Run Integration Tests
         run: echo TODO-TODO-TODO-TODO-TODO-TODO-TODO-TODO-TODO-TODO-TODO-TODO
```

Comment:
Should run `make test`

Reply:
Agreed this needed a bigger refactor, which is also in the works (discussion on slack)